reconcile/rhidp/ocm_oidc_idp/base.py

```diff
@@ -155,13 +163,17 @@ def act(
     ocm_map: OCMMap,
     current_state: Sequence[OCMOidcIdp],
     desired_state: Sequence[OCMOidcIdp],
+    managed_idps: list[str],
 ) -> None:
     """Compare current and desired OIDC identity providers and add, remove, or update them."""
     to_add = set(desired_state) - set(current_state)
     to_remove = set(current_state) - set(desired_state)
     to_compare = set(current_state) & set(desired_state)
 
     for idp in to_remove:
+        if managed_idps and idp.name not in managed_idps:
+            logging.debug(f"Skipping removal of unmanged '{idp.name}' IDP.")
```

Comment:
very important! typo: `unmanaaaaaged`

I am feeling so useful right now...

Reply:
It's `logging.debug`; no one will ever see this 🤦